dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.health-3.1.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.health-3.1
+singleton=true
+-features=io.openliberty.mpCompatible-4.0
+-bundles=io.openliberty.org.eclipse.microprofile.health.3.1; location:="dev/api/stable/,lib/"; mavenCoordinates="org.eclipse.microprofile.health:microprofile-health-api:3.1"
+kind=noship
+edition=core
```

Comment:
edition should be full

Reply:
@tevans78 Are all other MP features using `edition=core`?

Reply:
@tevans78 Made the update to switch it to `edition=core`. 
@fmhwong Tom had told me when using `kind=noship`, we have to use `edition=core`. However, when we switch it to either beta/ga, we also have to change the edition to `edition=core`
https://github.ibm.com/websphere/WS-CD-Open/wiki/Writing-features

Reply:
Updated.